task-list/app/controllers/tasks_controller.rb

```diff
@@ -2,6 +2,7 @@
 class TasksController < ApplicationController
 
   def index
+    @name = nil
```

Comment:
Rather than setting this variable explicitly to `nil` since it's used somewhere else, I think it would be better to check for `nil` within the view before attempting to display this variable